### PR TITLE
[PT] Minor spelling and phrasing fixes

### PR DESCRIPTION
**Pull request by lucassperez**
I was reading a lesson and found a few weird phrases in the portuguese version of the Documentation lesson

### CODE REVIEW

File: pt/lessons/basics/documentation.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Observe este script em Elixir (greeting.exs):
 IO.puts("Hello, " <> "chum.")
 ```
 
-Elixir, ao executar este script irá ignorar tudo, de `#` até o fim da linha, tratando-a como dados ocultos e sem lógica de execução. Pode adicionar nenhum valor para a operação ou o desempenho do script, no entanto, quando não é tão óbvio sobre o que está acontecendo, um programador deve saber ler o comentário. Esteja atento para não abusar do comentário de uma linha! Bagunçar uma base de código pode se tornar um pesadelo indesejável para alguns. É melhor usar com moderação.
+Elixir, ao executar este script, irá ignorar tudo de `#` até o fim da linha, tratando-a como dados ocultos e sem lógica de execução. Pode adicionar nenhum valor para a operação ou o desempenho do script, no entanto, quando não é tão óbvio o que está acontecendo, um programador deve saber ler o comentário. Esteja atento para não abusar do comentário de uma linha! Bagunçar uma base de código pode se tornar um pesadelo indesejável para alguns. É melhor usar com moderação.
```
Comment (thiamsantos) [inline pt/lessons/basics/documentation.md:31]:
```suggestion
Elixir, ao executar este script, irá ignorar tudo de `#` até o fim da linha, tratando-a como dados ocultos e sem lógica de execução. Pode adicionar nenhum valor para a operação ou alterar desempenho do script, no entanto, quando não é tão óbvio o que está acontecendo, um programador deve saber por ler o comentário. Esteja atento para não abusar do comentário de uma linha! Bagunçar uma base de código pode se tornar um pesadelo indesejável para alguns. É melhor usar com moderação.
```

:nail_care:

File: pt/lessons/basics/documentation.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.1.0
+version: 1.1.1
```
Comment (thiamsantos) [inline pt/lessons/basics/documentation.md:2]:
```suggestion
version: 1.1.0
```

We try to the version in sync with the english version. So we should only bump the when the english was also updated. In this case, as we are only fixing spelling and improve the lastest translation there's no need to update the version.

File: pt/lessons/basics/documentation.md (hunk 1 above)
Comment (lucassperez) [reply pt/lessons/basics/documentation.md:31]: Isn't "um programador deve saber **por** ler o comentário" a little weird? I don't think I have ever talked like that.

The other suggestion with the word "alterar" is really good! :+1:

File: pt/lessons/basics/documentation.md (hunk 2 above)
Comment (lucassperez) [reply pt/lessons/basics/documentation.md:2]: I didn't know that, thanks for letting me know! I'll put it back to `1.1.0` (:

File: pt/lessons/basics/documentation.md (hunk 1 above)
Comment (thiamsantos) [reply pt/lessons/basics/documentation.md:31]: Maybe: `quando não é tão óbvio o que está acontecendo, um programador deve conseguir entender ao ler o comentário.` 

WDYT?

Comment (lucassperez) [reply pt/lessons/basics/documentation.md:31]: I think this way it is good. I'll make the changes (:

**Review by RafaelCamarda — APPROVED** [review]

**Review by thiamsantos — APPROVED** [review]

**Pull request merged**